### PR TITLE
Fix send confirm email on SQL

### LINKED ISSUE #2517: Auth.form.error.email.invalid
**Informations**
- **Node.js version**: 10.x
- **NPM version**:  6.x
- **Strapi version**: v3.0.0-alpha.16
- **Database**: mongoDB
- **Operating system**: ubuntu

**What is the current behavior?**
I want to send confirmation email. ( with default SendMail provider)
I  enable email confirmation at `Roles & Permissions`
Send Post Request `localhost:1337/auth/local/register` with 

```
username
password
email
```
in result I got this 
```
{"statusCode":400,"error":"Bad Request","message":[{"messages":[{"id":"Auth.form.error.email.invalid"}]}]}
``` 
User was created but email was not send.

**Suggested solutions**
First I was thinking that is problem of default template of `Email address confirmation` where field `Response email` is empty. But it didn't help me.
 Also I tried to put emails in fields which are default empty here: Plugins>Email>configutation (Sendmail Default From and Sendmail Default Reply-To) - also didn't help me.

Btw in previous versions it was working fine. 
error comes from here:
/plugins/email/node_modules/strapi-provider-email-sendmail/lib/index.js

Code:
```
'use strict';

/**
 * Module dependencies
 */

// Public node modules.
const _ = require('lodash');
const sendmail = require('sendmail')({
  silent: true
});

/* eslint-disable no-unused-vars */
module.exports = {
  provider: 'sendmail',
  name: 'Sendmail',
  auth: {
    sendmail_default_from: {
      label: 'Sendmail Default From',
      type: 'text'
    },
    sendmail_default_replyto: {
      label: 'Sendmail Default Reply-To',
      type: 'text'
    }
  },
  init: (config) => {
    return {
      send: (options, cb) => {
        return new Promise((resolve, reject) => {
          // Default values.
          options = _.isObject(options) ? options : {};
          options.from = options.from || config.sendmail_default_from;
          options.replyTo = options.replyTo || config.sendmail_default_replyto;
          options.text = options.text || options.html;
          options.html = options.html || options.text;

          sendmail({
            from: options.from,
            to: options.to,
            replyTo: options.replyTo,
            subject: options.subject,
            text: options.text,
            html: options.html
          }, function (err) {
            if (err) {
              reject([{ messages: [{ id: 'Auth.form.error.email.invalid' }] }]);
            } else {
              resolve();
            }
          });
        });
      }
    };
  }
};

```

### PR DESCRIPTION
**Pull request by EpicUsaMan**
**My PR is a:**
- [ ] 💥 Breaking change
- [x] 🐛 Bug fix #2517
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

**Main update on the:**
- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [x] Plugin

**Manual testing done on the following databases:**
- [ ] Not applicable
- [ ] MongoDB
- [x] MySQL
- [ ] Postgres

**Description:**

Currently user doesn't have property email, but have attributes object with email property inside.

### CODE REVIEW

**Comment by lauriejim** [comment]
Hello @EpicUsaMan 
Can you give me the way to reproduce the issue you fix.
I don't see what you are trying to do.

**Comment by EpicUsaMan** [comment]
1. Set email confirmation in admin panel
2. Try to register new user via local provider (You can start strapi with debugger. Set breakpoint on line 317 in strapi-plugin-users-permissions/controllers/Auth.js
And check "user" variable in local variables. "user" (response from create method `const user = await strapi.query('user', 'users-permissions').create(params);`) variable doesn't have email field, but have this field in attributes)
3. Observe response
`{"statusCode":400,"error":"Bad Request","message":[{"messages":[{"id":"Auth.form.error.email.invalid"}]}]}`

**Comment by lauriejim** [comment]
Ho okay I see.

So I did what you explain

**1 -** Add the log in my file
screenshot 2019-02-06 at 12 30 23

**2 -** Display the log after making a register request
screenshot 2019-02-06 at 12 30 10

We can see my email is present.

Your issue is related to that point https://github.com/strapi/strapi/issues/2126

**Comment by lauriejim** [comment]
Ho okay it is with SQL database!

File: packages/strapi-plugin-users-permissions/controllers/Auth.js [hunk 1]
```diff
@@ -315,7 +315,7 @@ module.exports = {
         try {
           // Send an email to the user.
           await strapi.plugins['email'].services.email.send({
-            to: user.email,
+            to: user.attributes.email,
```
Comment (lauriejim) [inline packages/strapi-plugin-users-permissions/controllers/Auth.js]: Can you replace by `to: (user.toJSON ? user.toJSON() : user).email` please

Comment (EpicUsaMan) [reply packages/strapi-plugin-users-permissions/controllers/Auth.js]: Yes, thanks for example. I got it;)

**Comment by EpicUsaMan** [comment]
@lauriejim , hi.
I fix it.

**Pull request merged**